src/test/java/address/util/TestSerializationClass.java

```diff
@@ -0,0 +1,73 @@
+package address.util;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * A class used to test serialization and deserialization
+ */
+public class TestSerializationClass {
```

Comment:
I see. Probably the class name is misleading. When you start the name with 'Test' it sounds like a verb.